include/zenoh-pico/api/primitives.h

```diff
@@ -1492,6 +1492,18 @@ int8_t z_info_routers_zid(const z_loaned_session_t *zs, z_moved_closure_zid_t *c
  */
 z_id_t z_info_zid(const z_loaned_session_t *zs);
 
+/**
+ * Converts a Zenoh ID into a null-terminated string for print purposes.
```

Comment:
The string is no longer null-terminated, right ?

Reply:
Exact.